### PR TITLE
Updated SOA docs regarding Early access

### PR DESCRIPTION
**Pull request by sbattagi**
Hi,

We have reviewed the documentation in view of Early access for SOA customers and updated it with the latest information. Could you please review and approve this pull request?

NOTE: In additional resources section in the README.md file, we need to update the oracle blogs links (still work in progress by SOA Dev team). Once the blogs links are available we will update the same.

Thanks
Samba

### CODE REVIEW

**Comment by sbattagi** [comment]
Kindly add @rdas76  and @vivek-sam to this review. I don't see any option/button in the UI to add the reviewers on my own.

Thanks
Samba

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md [hunk 1]
```diff
@@ -41,6 +42,17 @@ Other than those considerations listed here, SOA Suite domains work in the same
 In this release, SOA Suite domains are supported using the “domain on a persistent volume”
 [model]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}) only, where the domain home is located in a persistent volume (PV).
 
+#### Prerequisites for SOA Suite Domains
+
+* Kubernetes 1.11.5+, 1.12.3+, 1.13.0+, 1.14.0+, and 1.15.0+ (check with `kubectl version`).
```
Comment (markxnelson) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: have you actually tested all of these?
i would *strongly* recommend that you limit to 1.13-1.15 and not go all the way back to 1.11
there are sec vuls on some of these too, so you cant go with the .0's - you need to check the cve's and pick dot releases that are safe

File: kubernetes/samples/scripts/create-soa-domain/README.md [hunk 2]
```diff
@@ -27,3 +27,10 @@ Refer to the [User guide](https://oracle.github.io/weblogic-kubernetes-operator/
 
 The fastest way to experience the operator is to follow the [User guide](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-fmw-domains/soa-suite/)
 or try out the deployment [scripts](https://oracle.github.io/weblogic-kubernetes-operator/samples/simple/domains/soa-domain/).
+
+### Additional Resources
```
Comment (markxnelson) [inline kubernetes/samples/scripts/create-soa-domain/README.md]: remove these - there are no links

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md [hunk 3]
```diff
@@ -15,6 +15,7 @@ it is supported for production use.
 #### Contents
 
 * [Introduction](#introduction)
+* [Prerequisites for SOA Suite Domains](#prerequisites-for-soa-suite-domains)
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: SOA Suite Domains -> SOA Suite domains

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md [hunk 4]
```diff
@@ -41,6 +42,17 @@ Other than those considerations listed here, SOA Suite domains work in the same
 In this release, SOA Suite domains are supported using the “domain on a persistent volume”
 [model]({{< relref "/userguide/managing-domains/choosing-a-model/_index.md" >}}) only, where the domain home is located in a persistent volume (PV).
 
+#### Prerequisites for SOA Suite Domains
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: SOA Suite Domains -> SOA Suite domains

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md [hunk 5]
```diff
@@ -60,14 +72,16 @@ following limitations currently exist for SOA Suite domains:
   currently supports the WebLogic MBean trees only.  Support for JRF MBeans has not
   been added yet.
 
+{{% notice note %}}
+For early access customers, with bundle patch access, it is recommended to build and use the Oracle SOA Suite Docker image with the latest bundle patches for Oracle SOA and Oracle Service Bus (OSB). The Oracle SOA Suite Docker image in `container-registry.oracle.com` does not have the bundle patches installed. However the customers who do not have access to the bundle patches can obtain the Oracle SOA Suite Docker image from `container-registry.oracle.com` as described in below section.
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: bundle patch access -> bundled patch access

Comment (rosemarymarano) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: it is recommended to build and use the Oracle SOA Suite Docker image with the latest bundle patches for Oracle SOA and Oracle Service Bus (OSB). -> we recommend that you build and use the Oracle SOA Suite Docker image with the latest bundled patches for Oracle SOA and Oracle Service Bus (OSB).

Comment (rosemarymarano) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: However the customers who do not have access to the bundle patches can obtain -> However, if you do not have access to the bundled patches, you can obtain

Comment (rosemarymarano) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: as described in below section. -> as described below.

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md [hunk 6]
```diff
@@ -84,22 +98,16 @@ Then, you can pull the image with this command:
...
-Additional information about using this image is available in the
-[Oracle Container Registry](https://container-registry.oracle.com).
-
 #### Creating a SOA Suite Docker image
 
-If you prefer, you can also create a Docker image containing the SOA Suite binaries yourself.
-A [sample](https://github.com/oracle/docker-images/tree/master/OracleSOASuite)
-is provided in the Oracle GitHub account that demonstrates how to create a Docker image
-to run SOA Suite.  
+You can also create a Docker image containing the Oracle SOA Suite binaries. This is the recommended approach for the customers who have access to Oracle SOA and Oracle Service Bus bundle patches.
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: This is the recommended approach for the customers who have access to Oracle SOA and Oracle Service Bus bundle patches. -> This is the recommended approach if you have access to Oracle SOA and Oracle Service Bus bundled patches.

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md [hunk 7]
```diff
@@ -84,22 +98,16 @@ Then, you can pull the image with this command:
...
 
-Please consult the [README](https://github.com/oracle/docker-images/blob/master/OracleSOASuite/dockerfiles/README.md) file associated with this sample for important prerequisite steps,
-such as building or pulling the Server JRE Docker image, Oracle FMW Infrastructure Docker image, and downloading the SOA Suite installer binary.
+Please consult the [README](https://github.com/oracle/docker-images/blob/master/OracleSOASuite/dockerfiles/README.md) file for important prerequisite steps,
+such as building or pulling the Server JRE Docker image, Oracle FMW Infrastructure Docker image, and downloading the Oracle SOA Suite installer and bundle patch binaries.
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: and bundle patch binaries -> and bundled patch binaries.

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md [hunk 8]
```diff
@@ -84,22 +98,16 @@ Then, you can pull the image with this command:
...
 
-You must also install the [required patch]({{< relref "/userguide/introduction/introduction/_index.md#prerequisites" >}})
-to use this image with the operator.  A [sample](https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure/samples/12213-patch-fmw-for-k8s)
-is provided that demonstrates how to create a Docker image with the necessary patch installed. Use this patched image for building the SOA Suite image.
+For the Fusion Middleware Infrastructure image you must install the [required patch]({{< relref "/userguide/introduction/introduction/_index.md#prerequisites" >}})
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: For the Fusion Middleware Infrastructure image you must -> For the Fusion Middleware Infrastructure image, you must
[required patch]({{< relref "/userguide/introduction/introduction/_index.md#prerequisites" >}}). (Put a period at the end of the sentence.)

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md [hunk 9]
```diff
@@ -84,22 +98,16 @@ Then, you can pull the image with this command:
...
+to use this image with the Oracle Weblogic operator.  A [sample](https://github.com/oracle/docker-images/tree/master/OracleFMWInfrastructure/samples/12213-patch-fmw-for-k8s)
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: Oracle Weblogic operator -> Oracle WebLogic Kubernetes Operator

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md [hunk 10]
```diff
@@ -129,10 +137,10 @@ the patch applied by running the provided script:
     ```
     $ docker tag oracle/fmw-infrastructure:12213-update-k8s oracle/fmw-infrastructure:12.2.1.3
     ```
-* Download the Oracle SOA Suite installer from Oracle Technology Network
-    or e-delivery.
+* Download the Oracle SOA Suite installer, latest Oracle SOA bundle patch (`30638100` or later) and Oracle Service Bus (OSB) bundle patch (`30059259` or later) from Oracle Technology Network or e-delivery.
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: bundle -> bundled

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md [hunk 11]
```diff
@@ -129,10 +137,10 @@ the patch applied by running the provided script:
...
+  >NOTE: Copy the installer binaries to the same location as the Dockerfile and the bundle patch zip files under the `docker-images/OracleSOASuite/dockerfiles/12.2.1.3/patches` folder.
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: bundle patch zip files -> bundled patch ZIP files

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md (hunk 1 above)
Comment (rdas76) [reply docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: Hi Mark, 

Thanks for the feedback, yes we should limit these to 1.13.0+ to 1.15.0+. Will also try to get the not dot release that has been tested. We will fix this. However, we pulled in these data from the FMW Infrastructure domain pre-req. Here is the link : https://oracle.github.io/weblogic-kubernetes-operator/userguide/introduction/introduction/

I think you should fix that too to remain consistent.

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md (hunk 3 above)
Comment (rdas76) [reply docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: This is title of the patch : 

BUG 30638100 - SOA BUNDLE PATCH 12.2.1.3.191208(ID:191208.1658.0113)

This is called "Bundle" not "Bundled".

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md (hunk 5 above)
Comment (rdas76) [reply docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: It's called bundle not bundled as mentioned above.

Comment (rdas76) [reply docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: Except bundle vs bundled okay to change.

Comment (rdas76) [reply docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: I would mention like this -

However, if you do not have access to the bundle patches, you can obtain the Oracle SOA Suite Docker image without bundle patches from `container-registry.oracle.com' as described in below section.

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md (hunk 3 above)
Comment (rosemarymarano) [reply docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: I stand corrected. Please ignore those edits.

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md (hunk 5 above)
Comment (rosemarymarano) [reply docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: Agreed.

Comment (rosemarymarano) [reply docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: I'm fine with that except the very last part of the sentence should be, "from `container-registry.oracle.com', as described below."

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md [hunk 12]
```diff
@@ -73,7 +73,7 @@ following limitations currently exist for SOA Suite domains:
   been added yet.
 
 {{% notice note %}}
-For early access customers, with bundle patch access, it is recommended to build and use the Oracle SOA Suite Docker image with the latest bundle patches for Oracle SOA and Oracle Service Bus (OSB). The Oracle SOA Suite Docker image in `container-registry.oracle.com` does not have the bundle patches installed. However the customers who do not have access to the bundle patches can obtain the Oracle SOA Suite Docker image from `container-registry.oracle.com` as described in below section.
+For early access customers, with bundle patch access, we recommend that you build and use the Oracle SOA Suite Docker image with the latest bundle patch for Oracle SOA. The Oracle SOA Suite Docker image in `container-registry.oracle.com` does not have the bundle patch installed. However, if you do not have access to the bundle patches, you can obtain the Oracle SOA Suite Docker image without bundle patches from `container-registry.oracle.com`, as described below.
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: if you do not have access to the bundle patches -> if you do not have access to the bundle patch
without bundle patches from -> without the bundle patch from

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md [hunk 13]
```diff
@@ -100,14 +100,12 @@ $ docker pull container-registry.oracle.com/middleware/soasuite:12.2.1.3
 
 #### Creating a SOA Suite Docker image
 
-You can also create a Docker image containing the Oracle SOA Suite binaries. This is the recommended approach for the customers who have access to Oracle SOA and Oracle Service Bus bundle patches.
+You can also create a Docker image containing the Oracle SOA Suite binaries. This is the recommended approach if you have access to Oracle SOA bundle patches.
```
Comment (rosemarymarano) [inline docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: access to Oracle SOA bundle patches. -> access to the Oracle SOA bundle patch.

File: docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md (hunk 1 above)
Comment (markxnelson) [reply docs-source/content/userguide/managing-fmw-domains/soa-suite/_index.md]: thanks @rdas76 - will do
@rjeberhard fyi -- our fmw infra list is using the .0's - we should fix that (or just point back to the main list)

**Pull request merged**